### PR TITLE
Moises.botarro/adding process agent process arg scrubbing

### PR DESCRIPTION
**Pull request by moisesbotarro**
@DataDog/burrito 

### What does this PR do?
Added a section under the Live Processes page to explain about the Process Arguments Scrubbing feature on the Process Agent 

### Motivation
We should make sure that the user knows that this feature is enabled by default and that he can expand it or disable it by changing some parameters on the configuration file. 

### Preview link
http://localhost:1313/graphing/infrastructure/process/#process-arguments-scrubbing

### CODE REVIEW

File: content/graphing/infrastructure/process.md [hunk 1]
```diff
@@ -92,6 +92,39 @@ In the [dd-agent.yaml][12] manifest used to create the daemonset, add the follow
 
 Refer to the standard [daemonset installation][13] and the [Docker Agent][11] information pages for further documentation.
 
+### Process Arguments Scrubbing
+
+In order to avoid that sensitive data is sent to Datadog servers and displayed on the Live Processes page, a Data Scrubber tool is shipped with the Process Agent. This feature is enabled by default and any process argument that matches one of the following words has its value hidden.
```
Comment (shang-wang) [inline content/graphing/infrastructure/process.md]: Change this to `In order to hide sensitive data on the Live Process page`

File: content/graphing/infrastructure/process.md [hunk 2]
```diff
@@ -92,6 +92,39 @@ In the [dd-agent.yaml][12] manifest used to create the daemonset, add the follow
...
+
+```
+"password", "passwd", "mysql_pwd", "access_token", "auth_token", "api_key", "apikey", "secret", "credentials", "stripetoken"
+```
+
+The matching process is case insensitive and to expand this list, use the `custom_sensitive_words` field on the `datadog.yaml` file. The user can also insert wildcards to widen the matching scope of a word. However, the use of a purely wildcard `*` is not supported.
```
Comment (shang-wang) [inline content/graphing/infrastructure/process.md]: Change to "Users can also use wildcards `*` to define their own matching scope. However, single wildcard `*` is not supported."

File: content/graphing/infrastructure/process.md (hunk 1 above)
Comment (conorbranagan) [reply content/graphing/infrastructure/process.md]: I would also rephrase "Data Scrubber tool" as it's not actually a separate tool. Maybe just `In order to hide... the process-agent will scrub arguments from the command line`

File: content/graphing/infrastructure/process.md (hunk 2 above)
Comment (shang-wang) [reply content/graphing/infrastructure/process.md]: Change to "The matching is case insensitive and user can define their own list, using `cutom_sensitive_words` field in `datadog.yaml` file under `process_config` section".

File: content/graphing/infrastructure/process.md [hunk 3]
```diff
@@ -92,6 +92,39 @@ In the [dd-agent.yaml][12] manifest used to create the daemonset, add the follow
...
+
+The  next image shows one process on the Live Processes page whose arguments have been hidden by using the configuration above.
+
+{{< img src="graphing/infrastructure/process/process_arg_scrubbing.png" alt="process arguments scrubbing" responsive="true" popup="true" style="width:100%;">}}
+
+Set `scrub_args` to `false` to completely disable the process arguments scrubbing.
+
+#### Agent 5
+
+The agent 5 also supports the scrubbing feature. To manage its behavior, use the following fields on the `datadog.conf` file under the `[process.config]` section:
```
Comment (conorbranagan) [inline content/graphing/infrastructure/process.md]: Agent should always be capitalized in our docs

File: content/graphing/infrastructure/process.md [hunk 4]
```diff
@@ -92,6 +92,39 @@ In the [dd-agent.yaml][12] manifest used to create the daemonset, add the follow
 
 Refer to the standard [daemonset installation][13] and the [Docker Agent][11] information pages for further documentation.
 
+### Process Arguments Scrubbing
+
+In order to hide sensitive data on the Live Process page, a Data Scrubber tool is shipped with the Process Agent. This feature is enabled by default and any process argument that matches one of the following words has its value hidden.
```
Comment (conorbranagan) [inline content/graphing/infrastructure/process.md]: > In order to hide sensitive data on the Live Processes page the Agent will scrub sensitive arguments from the process command line.

File: content/graphing/infrastructure/process.md [hunk 5]
```diff
@@ -92,6 +92,39 @@ In the [dd-agent.yaml][12] manifest used to create the daemonset, add the follow
...
+
+```
+"password", "passwd", "mysql_pwd", "access_token", "auth_token", "api_key", "apikey", "secret", "credentials", "stripetoken"
+```
+
+The matching is case insensitive and users can define their own list to be merged with the default one, using the `custom_sensitive_words` field in `datadog.yaml` file under `process_config` section. Users can also use wildcards * to define their own matching scope. However, single wildcard `*` is not supported.
```
Comment (conorbranagan) [inline content/graphing/infrastructure/process.md]: > wildcards (`*`) to define their own...
> However, a single wildcard (`*`) is not supported as a sensitive word.

**Comment by conorbranagan** [comment]
@l0k0ms Reverting this one, it's not available until we release 6.3. I'll re-open with a big note on it

**Comment by l0k0ms** [comment]
@conorbranagan Dully noted.
It's the merging rampage mode again. :) The LGTM lead me to the wrong conclusion.

**Pull request merged**